care/utils/tests/test_base.py

```diff
@@ -194,10 +236,12 @@ def setUpClass(cls) -> None:
         cls.super_user = cls.create_super_user(district=cls.district)
         cls.facility = cls.create_facility(cls.district)
         cls.patient = cls.create_patient()
+        cls.ambulance = cls.create_ambulance(district=cls.district, user=cls.user)
```

Comment:
Don't add ambulance create to base class. Others were added because all of them are used in almost all the cases. 
Ambulance will be used only for the ambulance APIs